inspectors/view/ui/inspector.cpp

```diff
@@ -214,6 +214,9 @@ void Inspector::update(Score* s)
                         case ElementType::REST:
                               ie = new InspectorRest(this);
                               break;
+                        case ElementType::MMREST:
+                              ie = new InspectorMMRest(this);
+                              break;
```

Comment:
This code is not used; `InspectorMMRest` does not exist, and this file has been deleted in https://github.com/musescore/MuseScore/commit/7a05ed77ba29562c132768958621c7dd633f1660#diff-e8bd3356faffc8c421f82c840e014162.